ivy-hydra.el

```diff
@@ -34,11 +34,11 @@
 (require 'hydra)
 
 (defun ivy--matcher-desc ()
-  "Return description of `ivy--regex-function'."
-  (if (eq ivy--regex-function
-          'ivy--regex-fuzzy)
-      "fuzzy"
-    "ivy"))
+  "Return description of `ivy--regex-function'. "
+  (let ((cell (assoc ivy--regex-function ivy--preferred-re-builders)))
+    (if cell
+        (cdr cell)
```

Comment:
> conceivably users might use this display name in other ways

By display name do you mean that given in `ivy--preferred-re-builders`, or the value returned by `ivy--matcher-desc`? In the latter case I would argue it's a non-user-facing function coupled with the hydra hint's formatting, and so a reasonable place to apply formatting transformations. Not that I have strong feelings about this - just confirming consensus.